modules/jaeger-config-storage.adoc

```diff
@@ -98,7 +98,7 @@ Minimum deployment =3
 |Number of central processing units for requests, based on your environment’s configuration.
 |Specified in cores or millicores (for example, 200m, 0.5, 1). For example, Proof of concept = 500m,
 Minimum deployment =1
-|1Gi
+|500m
 
```

Comment:
@JStickler Default values should be as follows,
```
CPU: 1
Memory: 16Gi
NodeCount: 3
```
![image](https://user-images.githubusercontent.com/1004403/115475995-7ba57a80-a25e-11eb-8142-01640d4dc796.png)

References:
* https://github.com/maistra/istio-operator/blob/53ae3883c096ca70278f918e0a118eb69a0603f8/resources/helm/v2.0/istio-telemetry/tracing/templates/jaeger-production-elasticsearch.yaml#L68~L69
* https://github.com/maistra/istio-operator/blob/53ae3883c096ca70278f918e0a118eb69a0603f8/resources/helm/v2.0/istio-telemetry/tracing/templates/jaeger-production-elasticsearch.yaml#L48